requirements.txt

```diff
@@ -1,38 +1,55 @@
 # For outdated module report see https://requires.io/github/OpenTreeMap/otm-core/requirements/?branch=develop
 # Changelog URL shown below if not linked on above page
 
-boto==2.42.0                             # http://docs.pythonboto.org/en/latest/releasenotes/v2.39.0.html
-# https://docs.djangoproject.com/en/1.10/releases/#id2
-Django==1.11.3  # rq.filter: >=1.11,<1.12
-django-apptemplates==1.2
+amqp==1.4.9
+anyjson==0.3.3
+billiard==3.3.0.23
+boto==2.48.0                             # http://docs.pythonboto.org/en/latest/releasenotes/v2.39.0.html
 # django-celery requires celery, but is incompatible with celery
 # 4.0. django-celery has added this restriction on the master branch,
 # but it has not yet been released.
 celery>=3.1.15,<4.0
```

Comment:
Fixed in (rebased) commit https://github.com/OpenTreeMap/otm-core/pull/3184/commits/a971d4312fe45b5e44cd05986f6e777385cf8d50